react-components/src/util/Autocomplete.js

```diff
@@ -0,0 +1,107 @@
+import MenuItem from "@material-ui/core/MenuItem";
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import Select from "react-select";
+import { withStyles } from "@material-ui/core/styles";
+
+/**
+ * @author aramsey
+ * Renders the Select component from react-select with some stylings for material-ui
+ */
+
+// Copied from https://material-ui.com/demos/autocomplete/
+const styles = theme => ({
+    // We had to use a lot of global selectors in order to style react-select.
+    // We are waiting on https://github.com/JedWatson/react-select/issues/1679
+    // to provide a much better implementation.
+    // Also, we had to reset the default style injected by the library.
+    '@global': {
+        '.Select-input input': {
+            cursor: 'default',
+            display: 'block',
+            fontFamily: 'inherit',
+            fontSize: 'inherit',
+            margin: 0,
+            outline: 0,
+            width: '100% !important',
+        },
+        '.Select-menu-outer': {
+            backgroundColor: theme.palette.background.paper,
+            boxShadow: theme.shadows[2],
+            width: '100%',
+            zIndex: 2,
+            maxHeight: 125,
+        },
+        '.Select-menu': {
+            maxHeight: 125,
+            overflowY: 'auto',
+        },
+
+    }
+});
+
+function Autocomplete(props) {
+    let {
+        variant,
+        ...custom
+    } = props;
+
+    let Component = null;
```

Comment:
I might have named this to something that's not already in use (`SelectComponent`?), but since it's a local `let`, it's not a big deal either 🤷‍♂️

Reply:
Sure, I'll rename it before merging 👍